pkgs/top-level/python-packages.nix

```diff
@@ -28490,15 +28490,25 @@ in modules // {
 
   mps-youtube = buildPythonPackage rec {
     name = "mps-youtube-${version}";
-    version = "0.2.6";
+    version = "0.2.7.1";
 
     disabled = (!isPy3k);
 
+    # disabled due to error in loading unittest
+    # don't know how to make test from: <mps_youtube. ...>
+    doCheck = false;
+
+    # before check create a directory and redirect XDG_CONFIG_HOME to it
```

Comment:
The comment should make clear why this is needed. Stating what the 4 lines below do adds no value if they are trivial.
Also is this even executed with doCheck = false?